packages/forms-web-app/src/routes/full-appeal/submit-appeal/index.js

```diff
@@ -25,6 +25,8 @@ const healthSafetyIssuesRouter = require('./health-safety-issues');
 const identifyingTheOwners = require('./identifying-the-owners');
 const plansDrawingsRouter = require('./plans-drawings');
 const supportingDocumentsRouter = require('./supporting-documents');
+const tellingTheLandowners = require('./telling-the-landowners');
+const tellingTheTenants = require('./telling-the-tenants');
```

Comment:
Could you please add `Router` to the end of these variables to keep consistency with the others?